release_test_compare.py

```diff
@@ -436,6 +449,14 @@ def compare_consensus(sample: Sample,
             has_big_change = False
             dummy_row = {'refseq.nuc.pos': '-1',
                          'coverage': '0'}
+            if MICALL_VERSION == '7.15':
+                if region.split('-')[-1] == 'NS5b':
+                    target_nucs = [nuc for nuc, row in target_details]
+                    last_codon = ''
+                    for nuc in target_nucs[-3:-1]:
```

Comment:
Are you sure about this? I think it will only give you two characters. You probably meant `target_nucs[-3:]`.

Reply:
Yes, thanks for spotting that.